_partials/self-hosted/_setup-steps.mdx

```diff
@@ -0,0 +1,243 @@
+    ```
+
+14. Reboot your RHEL VM to apply the changes.
+
+    ```shell
+    reboot
+    ```
+
+    <!-- prettier-ignore -->
+    Your SSH connection will be terminated. Start a new SSH session and switch to `sudo` mode before proceeding. Use
+    the same commands from steps **5**, **6**, and **12** of this guide.
+
+15. Issue the following command to configure the Harbor registry. Replace `palette.example.com` with the FQDN of the
+    RHEL airgap VM. The script will generate a self-signed certificate for the value you provide.
+
+    ```shell
+    /bin/airgap-setup.sh palette.example.com
+    ```
+
+    The script output should look similar to the example below. It contains the credentials and values you will need when
+    completing the installation with the Palette CLI. If you need to review this information in the future, invoke the
+    script again.
+
+    ```text hideClipboard
+    Setting up SSL Certs
+    /opt/spectro/functions.sh: line 118: /etc/nginx/.htpasswd: No such file or directory
+    chmod: cannot access '/etc/nginx/.htpasswd': No such file or directory
+    mkdir: cannot create directory ‘/etc/nginx/ssl’: No such file or directory
+    cp: target '/etc/nginx/ssl' is not a directory
+    Setting up Harbor
+    setenforce is /usr/sbin/setenforce
+    Setup Completed
+
+    Details:
+    -------
+    Spectro Cloud Repository
+    Location: https://palette.example.com:8443
+    UserName: spectro
+    Password: **************
+    CA certificate filepath: /opt/spectro/ssl/server.crt
+
+    Pack OCI Registry
+    Endpoint: https://palette.example.com
+    Base Content Path: spectro-packs
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+
+    Image OCI Registry
+    Endpoint: https://palette.example.comv
+    Base Content Path: spectro-images
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+    ```
+
+16. Update the SSL certificate file and key in the **httpd** service.
+
+    :::warning
+
+    You can skip this step if you do not plan to use the local Spectro Cloud Artifact Repository (SCAR) during the
+    [{props.edition} installation](#next-steps) process.
+
+    :::
+
+    Open the `/etc/httpd/conf.d/ssl.conf` file and add the path to the certificate and key generated in step **15** of
+    this guide:
+
+    - Replace the line `SSLCertificateFile /etc/pki/tls/certs/localhost.crt` with
+      `SSLCertificateFile  /opt/spectro/ssl/server.crt`.
+    - Replace the line `SSLCertificateKeyFile /etc/pki/tls/private/localhost.key` with
+      `SSLCertificateKeyFile /opt/spectro/ssl/server.key`.
+
+    Save and exit the file. Next, restart the HTTP server to apply the changes.
+
+    ```shell
+    systemctl restart httpd.service
+    ```
+
+17. Start the {props.edition} installation binary, which uploads release-specific {props.edition} images and packs to the Harbor registry.
```

Comment:
Same as above, just tell them to reference the filename that includes the version number. 
Something like, "Replace the filename with the file you received from your Palette support representative" or something like that 😄